index.html

```diff
@@ -60,43 +71,51 @@ <h3>LET'S CONNECT</h3>
       </section>
     </div>
     <!-- ProjectsSection -->
-    <section class="projectsContainer">
+    <section class="projectsContainer" id="projects">
         <div class="project1">
```

Comment:
- [ ] Please consider switching the direction of the property flex for the project card so the content layout won't overlap as shown in the below screenshot 👍🏼 

### Your Desing 
![image](https://user-images.githubusercontent.com/68030297/230244154-2c1309d6-6836-43ce-b07f-0ae256ca2dc5.png)